spec/workers/workflows_dump_worker_spec.rb

```diff
@@ -20,7 +20,7 @@
         tasks: tasks, pairwise: !workflow.pairwise,
         grouped: !workflow.grouped, prioritized: !workflow.prioritized
       }
-      workflow.update_attributes(updates)
+      workflow.update(updates)
```

Comment:
did this one sneak into this PR? happy for it to stay

Reply:
oops very possible that i lumped it in since i was on the spec file anyway.